scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -153,7 +153,21 @@ trait DocumentOps { self: DatabaseOps =>
               // https://github.com/scalameta/scalameta/issues/665
               // Instead of crashing with "unsupported file", we ignore these cases.
               if (gsym0 == null) return
-              if (gsym0.isAnonymousClass) return
+              if (gsym0.isAnonymousClass) {
+                if (config.members.isAll) {
+                  gsym0.tpe.parents
```

Comment:
`gsym0.asClass.parentSymbols`

Reply:
What is the difference?

Reply:
oh I see, I don't need to `map typeSymbol`